sim/battle-actions.ts

```diff
@@ -993,7 +993,7 @@ export class BattleActions {
 		if (move.target === 'all' && !isSelf) {
 			hitResult = this.battle.singleEvent('TryHitField', moveData, {}, target || null, pokemon, move);
 		} else if ((move.target === 'foeSide' || move.target === 'allySide' || move.target === 'allyTeam') && !isSelf) {
-			hitResult = this.battle.singleEvent('TryHitSide', moveData, {}, target || null, pokemon, move);
+			hitResult = this.battle.singleEvent('TryHitSide', moveData, {}, target && target.side || null, pokemon, move);
```

Comment:
```suggestion
			hitResult = this.battle.singleEvent('TryHitSide', moveData, {}, target?.side || null, pokemon, move);
```

Reply:
Why does this function take a `side`? The `runEvent` version takes a `Pokemon`, and I don't think there are any consumers of this as a single event anyway.